### PR TITLE
Add VFI-Mamba as library

### PR DESCRIPTION
**Pull request by NielsRogge**
This PR is linked with https://github.com/MCG-NJU/VFIMamba/pull/2 to ensure download stats are working for the VFI-Mamba repo.

It also adds a "How to use this model" button with a code snippet.

### CODE REVIEW

**Review by Wauplin — COMMENTED** [review]
Thanks for the PR @NielsRogge!

Let's wait for the integration to be 100% settled and to have already a few models in https://huggingface.co/models?other=vfi-mamba and then we will be able to merge

File: packages/tasks/src/model-libraries-snippets.ts [hunk 1]
```diff
@@ -871,6 +871,13 @@ IWorker engine = WorkerFactory.CreateWorker(BackendType.GPUCompute, model);
 `,
 ];
 
+export const vfimamba = (model: ModelData): string[] => [
+	`from Trainer_finetune import Model
+
+model = Model(-1)
+model.from_pretrained("${model.id}")`,
```
[the remark this answers is not in the document]
Comment (Wauplin) [reply packages/tasks/src/model-libraries-snippets.ts:878]: New snippet is

```suggestion
model = Model.from_pretrained("${model.id}")`,
```


now that https://github.com/MCG-NJU/VFIMamba/pull/2#discussion_r1741124993 has been merged

**Review by Wauplin — APPROVED** [review]
Look good to me! PR can be merged as soon as there are repos listed in https://huggingface.co/models?other=vfi-mamba

I just opened https://huggingface.co/MCG-NJU/VFIMamba_S/discussions/1 and https://huggingface.co/MCG-NJU/VFIMamba/discussions/1.

**Comment by Wauplin** [comment]
PRs are merged so we can merged this one now.

**Review by bcvladilav — APPROVED** [review]

**Review by MrAungThuHtet — APPROVED** [review]

**Review by Mandorenix — APPROVED** [review]

**Review by Gtk-thailand — APPROVED** [review]

**Review by gerax222 — COMMENTED** [review]
R

**Review by ferdragonf-spec — APPROVED** [review]

**Pull request merged**